lib/commands/network.js

```diff
@@ -76,13 +76,9 @@ commands.toggleFlightMode = async function () {
   });
 };
 
-commands.setGeoLocation = async function (location) {
-  /*
-   * TODO: Implement isRealDevice(). This method fails on
-   * real devices, it should throw a NotYetImplementedError
-   */
-  let cmd = `geo fix ${location.longitude} ${location.latitude}`;
-  return await this.adb.sendTelnetCommand(cmd);
+commands.setGeoLocation = async function(location) {    
```

Comment:
A space is needed after `function` and before the opening parenthesis (see https://github.com/appium/appium/blob/1.5/docs/en/contributing-to-appium/style-guide-2.0.md#style-notes).